pages/chat/index.vue

```diff
@@ -1,8 +1,8 @@
 <template>
   <div>
-    <THeader title="Chat" />
+    <THeader :title="$t('chat.title')" />
     <div v-if="!chats" class="p-16 text-xs text-center">
-      Start conversations and they will appear here.
+      {{ $t('chat.conversation.new') }}
```

Comment:
I would call this one also `empty`

```suggestion
      {{ $t('chat.empty') }}
```